src/main/java/nitezh/ministock/activities/GlobalWidgetData.java

```diff
@@ -23,4 +40,187 @@
     public void setGlobalList(List<WidgetRow> list) {
         myStockList = list;
     }
+
+    // Variables in Graph Implementation
+    public void setURLString(String urlString) {
+
+        this.urlString = urlString;
+    }
+
+    public String getURLString() {
+
+        return urlString;
+    }
+
+
+
+    // Used in Time Scale of Graph Implementation
+    public static void setInterval(int intervalVal) {
+
+        interval = intervalVal;
+    }
+
+    public static int getInterval() {
+        return interval;
+    }
+
+
+    // Methods for Graph Implementation
+    public static JsonObject getJsonObjectRoot(String sURL) {
+        JsonObject rootObj;
```

Comment:
`public static JsonObject getJsonObjectRoot(String sURL)`
method never used and should be removed if we are not going to use it in the future

Reply:
I'll keep this in for future use.